test/stdlib/Span/InlineSpanProperties.swift

```diff
@@ -119,6 +119,20 @@ suite.test("InlineArray.span property (String)")
   }    
 }
 
+suite.test("InlineArray.span property (empty, aligned)")
+.skip(.custom(
+  { if #available(SwiftStdlib 6.2, *) { false } else { true } },
+  reason: "Requires Swift 6.2's standard library"
+))
+.code {
```

Comment:
```suggestion
.require(.stdlib_6_2).code {
```
There's no reason to use the multi-line version now that we have `.require(.stdlib_M_N)`. Many tests that predated it simply haven't been updated.